sw/include/params.h

```diff
@@ -51,3 +51,12 @@ static const uint64_t __BOOT_DTB_TYPE_GUID[2] = {0x42DE2AEFBA442F61UL, 0x9DCB3A5
 
 // GUID of firmware partition we boot into
 static const uint64_t __BOOT_FW_TYPE_GUID[2] = {0x4B0D3F5B99EC86DAUL, 0x59F8A5CFBAC44B8FUL};
+
+// IDs for AXI-REALM managers
+// Adjust if more CVA6 cores, default to 1
+enum axirealm_mngr_id {
+    AXIREALM_MNGR_ID_CVA60 = 0,
+    AXIREALM_MNGR_ID_DBG = 1,
+    AXIREALM_MNGR_ID_DMA = 2,
+    AXIREALM_MNGR_ID_SL = 3
+};
```

Comment:
We shouldn't hardcode things we know are parameterization-specific here.

Instead, you can read the `num_int_harts` register to know how many internal harts there are and `hw_features` to know which hardware features are available.

Reply:
Makes sense, I already use the `num_int_harts` in the test. Will fix ASAP.

Reply:
done